doc/genmods.py

```diff
@@ -60,7 +60,6 @@
     ('probe', 'Instrument probe'),
     ('profile', 'Model profile'),
     ('reflectivity', 'Reflectivity'),
-    ('reflmodule', 'Low level reflectivity calculations'),
```

Comment:
@pkienzle - do I need to re-enable this (with a different name) with some documentation for the low-level calculations?  I'm not quite sure how to do that.

Reply:
I think I figured it out...